subscription_oca/security/ir_rule.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="sale_subscription_comp_rule" model="ir.rule">
+        <field name="name">Sale Subscription</field>
+        <field name="model_id" ref="model_sale_subscription" />
+        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

Comment:
Could you please use a more common and legit representation that you could find in odoo core : 
`	['|',('company_id','=',False),('company_id','in',company_ids)]`